### ISSUE TITLE
Conduct manual accessibility checks for all relevant components

### ISSUE THREAD

**Issue by bengolder** [codeforamerica/honeycrisp-gem#291]
### Summary

For the components listed below, do a manual accessibility check and [document](https://airtable.com/shrzLthzu1DBWEtYd) whether the
component passes those tests or has issues.

### Details
- Timebox to 1 day at most. Anything remaining can be marked as untested.
- The [database](https://airtable.com/tblqitV9vGwdutRaq/viwV4COl82NqbMoTE?blocks=hide) and [form](https://airtable.com/shrzLthzu1DBWEtYd) are one approach I thought might make this easier to document. Let me know if you have any feedback about how documentation or tracking could be easier.
- [Reveal: manual accessibility checks #280](https://github.com/codeforamerica/honeycrisp-gem/issues/280)
- Steps for manual checks
  - review the component using the WAVE browser plugin
  - interact with the component without using a mouse (only a keyboard)
    - you should be able to fully use the [component with just a keyboard](https://webaim.org/techniques/keyboard/)
    - focus should clearly visible when interacting via keyboard
  - optional additional checks
    - examine touch target size, make sure [it's not too small (maybe 35x35)](https://www.w3.org/WAI/WCAG21/quickref/#qr-target-size)
    - use [assistiv labs](https://assistivlabs.com/) (shared login in lastpass, if it's not there, contact Alex or Ben S)
      - check windows high contrast mode and make sure that keyboard navigation and focus look good
      - interact with the component using NVDA screen reader on windows
    - interact with component using VoiceOver on iPhone

#### Components

_This is a preliminary list. Please feel free to add, remove, or give feedback about the inclusion of any components in this list._

- [x] [Accordion](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-accordion)
- [x] [Auto-formatted Input](http://honeycrisp.herokuapp.com/cfa/styleguide#atoms-form_elements)
- [x] Document upload button
- [x] [Follow up question](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-follow_up_question)
- [x] [Reveal](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-reveal)
- [x] [Buttons](http://honeycrisp.herokuapp.com/cfa/styleguide#atoms-buttons)
- [x] [Checkboxes](http://honeycrisp.herokuapp.com/cfa/styleguide#atoms-form_elements)
- [x] [Data table](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-data_table)
- [x] Date select
- [x] Pagination
- [x] [Progress Indicator](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-progress_indicator)
- [x] [Progress step bar](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-progress_step_bar)
- [x] Radio buttons
- [x] [Searchbar](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-searchbar)
- [x] Select
- [x] [Steps](http://honeycrisp.herokuapp.com/cfa/styleguide#organisms-steps)
- [x] [Summary table](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-summary_table)
- [x] [Tab bar](http://honeycrisp.herokuapp.com/cfa/styleguide#molecules-tab_bar)
- [x] Text input
- [x] Textarea
- [x] [Vertical steps](http://honeycrisp.herokuapp.com/cfa/styleguide#organisms-vertical_steps)

### Acceptance

This task is complete when all of the listed components have a documented result for the specific Inclusivity Score control.
You can document the result [here](https://airtable.com/tblqitV9vGwdutRaq/viwV4COl82NqbMoTE?blocks=hide) using [this form](https://airtable.com/shrzLthzu1DBWEtYd).

**Comment by coltborg** [comment]
Complete
